recipe-app/View/RecipeDetailViewController.swift

```diff
@@ -0,0 +1,183 @@
+        let analyzedInstructions = recipeDetailViewModel.recipeDetail.analyzedInstructions
+        for step in analyzedInstructions[0].steps {
+            instructionsLabel.text = "\(instructionsLabel.text ?? "") Step \(step.number): \(step.step)\n\n"
+        }
+        
+        instructionsLabel.backgroundColor = .white
+        instructionsLabel.font = UIFont(name: "HelveticaNeue", size: 20)
+        instructionsLabel.translatesAutoresizingMaskIntoConstraints = false
+        instructionsLabel.numberOfLines = 0
+        instructionsLabel.sizeToFit()
+    }
+    
+    private func configureConstraints() {
+        NSLayoutConstraint.activate([
+            
+            titleLabel.topAnchor.constraint(equalTo: contentView.safeAreaLayoutGuide.topAnchor, constant: 20),
+            titleLabel.bottomAnchor.constraint(equalTo: image.topAnchor, constant: -30),
+            titleLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            
+            image.topAnchor.constraint(equalTo: contentView.safeAreaLayoutGuide.topAnchor, constant: 100),
+            image.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            image.widthAnchor.constraint(equalToConstant: 350),
+            image.heightAnchor.constraint(equalToConstant: 300),
+            
+            ingredientsLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            ingredientsLabel.topAnchor.constraint(equalTo: image.bottomAnchor, constant: 20),
+            ingredientsLabel.widthAnchor.constraint(equalToConstant: 380),
+            ingredientsLabel.bottomAnchor.constraint(equalTo: instructionsLabel.topAnchor, constant: 50),
+            
+            instructionsLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            instructionsLabel.topAnchor.constraint(equalTo: ingredientsLabel.bottomAnchor, constant: 30),
+            instructionsLabel.widthAnchor.constraint(equalToConstant: 380),
+            instructionsLabel.bottomAnchor.constraint(equalTo: contentView.safeAreaLayoutGuide.bottomAnchor, constant: -10),
+            
+            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            
+            contentView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            contentView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            contentView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            contentView.heightAnchor.constraint(equalTo: scrollView.heightAnchor, multiplier: 3),
+            contentView.widthAnchor.constraint(equalTo: scrollView.widthAnchor)
+        ])
+    }
+}
+
+extension UIImageView {
```

Comment:
We can put this in a extensions folder with its own file.

eg. UIImageView+Extensions.swift